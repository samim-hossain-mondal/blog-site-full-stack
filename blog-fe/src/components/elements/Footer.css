.footer {
  margin-top: auto;
  bottom: 0;
  width: 100%;
  display: flex;
  align-content: center;
  color: rgb(255, 255, 255);
  background-color: #000000;
}

.footer>p {
  padding: 10px;
  padding-left: 135px;
}