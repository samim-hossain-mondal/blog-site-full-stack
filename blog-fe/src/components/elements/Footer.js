import React from 'react';
import './Footer.css';

export default function Footer() {
  return (
    <div className='footer'>
      <p>&copy;arifact.com 2019</p>
    </div>
  );
}