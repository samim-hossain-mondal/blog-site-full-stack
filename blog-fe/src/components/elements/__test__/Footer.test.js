import React from "react";
import { render } from "@testing-library/react";

import Footer from "../Footer";

describe("Footer", () => {
  it("should render", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});