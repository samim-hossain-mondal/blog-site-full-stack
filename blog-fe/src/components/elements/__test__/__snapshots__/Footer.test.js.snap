// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <p>
      Â©arifact.com 2019
    </p>
  </div>
</DocumentFragment>
`;
